src/map/atcommand.c

```diff
@@ -4100,6 +4100,8 @@ ACMD(mapinfo)
 		strcat(atcmd_output, msg_fd(fd, 1292)); // PrivateAirshipStartable |
 	if (map->list[m_id].flag.pairship_endable)
 		strcat(atcmd_output, msg_fd(fd, 1293)); // PrivateAirshipEndable |
+	if (map->list[m_id].flag.nopet != 0)
```

Comment:
here and in clif_sendegg you do different checks here ``!= 0`` and in clif_sendegg `` == 1``
probably need do same check in both places?

Reply:
fixed